target/target_test.go

```diff
@@ -1,63 +1,37 @@
-package target
+package target_test
 
 import (
-	"io/ioutil"
-	"os"
 	"testing"
-	"time"
+
+	"github.com/magefile/mage/target"
 )
 
 func TestPath(t *testing.T) {
-	t.Parallel()
-	dirs := []string{"testdata/src_dir", "testdata/target_dir"}
-	for _, d := range dirs {
-		os.MkdirAll(d, 0777)
-		time.Sleep(10 * time.Millisecond)
-	}
-	files := []string{
-		"testdata/src_dir/some_file",
-		"testdata/src_dir/some_",
-		"testdata/src_file",
-		"testdata/target_dir/some_file",
-		"testdata/target_file",
-	}
-	for _, v := range files {
-		err := ioutil.WriteFile(v, []byte(v), 0600)
-		if err != nil {
-			t.Fatal(err)
-		}
-		time.Sleep(500 * time.Millisecond)
-	}
-	defer func() {
-		err := os.RemoveAll("testdata")
-		if err != nil {
-			t.Fatal(err)
-		}
-	}()
-
-	table := []struct {
-		desc    string
-		src     string
-		targets []string
-		expect  bool
+	tests := []struct {
+		name    string
+		dst     string
```

Comment:
The _target_ of the `target package` is called _dst_ by the API. This may be clearer if the first argument where _tgt_.